contrib/ci-horizen/scripts/build/build_amd64_linux_ubuntu_jammy.sh

```diff
@@ -5,4 +5,4 @@ set -euo pipefail
 docker run --rm -v "${TRAVIS_BUILD_DIR}:/mnt/build" -v "$HOME/.ccache:/mnt/.ccache" -e LOCAL_USER_ID="$(id -u)" \
   -e LOCAL_GRP_ID="$(id -g)" --env-file <(env | grep 'DOCKER_\|MAKEFLAGS') "${IMAGE_NAME}:${IMAGE_TAG}" \
   bash -c 'set -xeuo pipefail && export HOST="$(gcc -dumpmachine)" && export MAKEFLAGS="${MAKEFLAGS:-} -j $(($(nproc)+1))" \
-    && cd "${DOCKER_HOME}" && /usr/bin/time -v ./zcutil/build.sh $MAKEFLAGS'
+    && cd "${DOCKER_HOME}" && bear -- ./zcutil/build.sh $MAKEFLAGS'
```

Comment:
A couple of questions here:
- Can we keep `/usr/bin/time -v` before `bear` in order to have statistics about the build process?
- What is the purpose of `--`? I couldn't find any mention in the help.

Reply:
For the first remark, I updated the script as suggested.
As for the `--` option of `bear`, it is required for versions > 2.4, as those installed by default on Ubuntu Jammy through `apt` package manager (see https://github.com/rizsotto/Bear#how-to-use )

Reply:
I was using version 2.4, that's why I couldn't find it. Thanks a lot!